plugins/plugin-dotenv/test/plugin.test.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+const execa = require('execa');
+
+const execPluginFilePath = require.resolve('./execPlugin');
+const testWorkDirectory = path.join(__dirname, 'env');
+
+function execPlugin(nodeEnv) {
+  const {stdout} = execa.sync('node', [execPluginFilePath], {
+    cwd: testWorkDirectory,
+    env: {NODE_ENV: nodeEnv},
+  });
+  return JSON.parse(stdout);
+}
+
+beforeEach(() => {
+  process.env.__DOTENV_PRESET = 'PRESET';
```

Comment:
Could you please explain why you override the `__DOTENV_PRESET` environment variable here? I'm new to this plugin, apologies if I miss something obvious

Reply:
this plugin should not override the environment variables that already exist. The `__DOTENV_PRESET` cover this situation.

Reply:
Ah, thank you! Could you add a comment with an explanation above?

Maybe something like this?

```suggestion
  // test that pre-exiting environment variables are not overwritten
  process.env.__DOTENV_PRESET = 'PRESET';
```

Reply:
nice, added the comment.